site/contents/write_ups/2022/09-03-22_writeup.md

```diff
@@ -0,0 +1,95 @@
+
+We were briefly sidetracked by the relative weakness of the speaker's definition of autonomy in comparison to the definition as it relates to humans. With humans, autonomy is more than just the ability identify and attack a target. With humans, morale is a factor whereas with autonomous robots, this is not a factor. Perhaps this is a good thing? Defining what autonomous means "the AI chooses who to target" could also apply to propaganda/advertising as well as weapon targeting. 
+
+Each new development in technological warfare is supposed to "end war". Is this just another weapon which can have more disastrous outcomes? What is going to be the effect of using these weapons? It will target soldiers better than targetting civilians. Wars tend to only stop when the number of losses is great enough. If they're not humans, surely these systems won't have morality, won't get tired and don't need motivation and so might wage more unethical wars. This leads to a problem of AI allignment and what objectives are coded into the AI.
+
+We've not even considered the ethics of war, let alone different types of it. Can we really say one type of war is better than another? That one type is more or less ethical? Theorising about war boils down to deducing the conditions by which it is okay to kill someone. 
+
+Is the inclusion of AI into this discussion actually interesting or relevant?
+
+### In your opinion, is there a valid argument that lethal autonomous weapons could reduce negative outcomes? Is this different for civilians versus soldiers?
+
+It's clear that several things that are wrong with it. Most people wars only stop when enough people are killed. 
+
+Will autonomous weapons be able to distinguish/discriminate people? What about so called grey zone conflicts? There are a lot of areas where you don't really know if its a conflict. Categorising anything qualitative is problematic, but here the stakes are considerably higher. Is someone carrying a gun a soldier? A knife? Will the robots detect armbands correctly? Maybe we'll end up using our fitbits to declare which side we're on. If drones were accurate at telling the difference between military and civilians, that could be a positive thing. But civilians also take up arms. This would change warfare - military would wear civilian clothes (you also see civilias wearing camo). Humans are bad at distinguishing between civilians and military targets. Is it better to have a human behind it so that at least they might feel some guilt if something goes wrong?
+
+Putting anything into numbers runs into this problem. This isn't an optimisationg problem, this is life or death. What training sets of data for machine learning systems? How do you come up with performance metrics? You almost encode some sort of morality by setting objective functions. It will lead to very imbalanced wars. Again we fall onto the problem of trying to decide which wars are more or less ethical.
```

Comment:
```suggestion
Putting anything into numbers runs into this problem. This isn't an optimisation problem, this is life or death. What training sets of data are used for these machine learning systems? How do you come up with performance metrics? You almost encode some sort of morality by setting objective functions. It will lead to very imbalanced wars. Again we fall onto the problem of trying to decide which wars are more or less ethical.
```